alica_engine/src/engine/PlanBase.cpp

```diff
@@ -39,11 +40,16 @@ PlanBase::PlanBase(AlicaEngine* ae)
         , _isWaiting(false)
 
 {
-    essentials::SystemConfig& sc = essentials::SystemConfig::getInstance();
+    std::function<void(const YAML::Node& config)> reloadFunctionPtr = std::bind(&PlanBase::reload, this, std::placeholders::_1);
```

Comment:
in this case you can replace "std::function<void(const YAML::Node& config)> " with auto